lib/google_drive/api_client_fetcher.rb

```diff
@@ -22,9 +22,10 @@ def initialize(authorization)
       @drive.authorization = authorization
       # Make the timeout virtually infinite because some of the operations (e.g., uploading a large file)
       # can take very long.
-      @drive.client_options.open_timeout_sec = 100_000_000
-      @drive.client_options.read_timeout_sec = 100_000_000
-      @drive.client_options.send_timeout_sec = 100_000_000
+      t = (2**31 - 1) / 1000
```

Comment:
Can you mention that in a comment in the code to remind developers (mainly me) where that number comes from? Also mention that it's a restriction of JRuby, instead of Google API? Looks like the issue is specific to JRuby because it's not an issue in CRuby.